app/components/ui/molecules/BurgerMenu.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+import Icon from '../atoms/Icon'
+import ButtonAsIconWrapper from '../atoms/ButtonAsIconWrapper'
+import CrossIconButton from '../atoms/CrossIconButton'
+import NavLink from '../../ui/atoms/NavLink'
+import ModalOverlay from '../molecules/ModalOverlay'
+
+const MenuPanel = styled.div`
+    max-width: 240px;
+    background-color: ${th('colorBackground')}
+    height: 100%;
+`
+
+const MenuItem = styled(Box)`
+  border-top: ${th('borderWidth')} ${th('borderStyle')} ${th('colorBorder')};
+  height: ${th('space.6')};
+  &:first-child {
+    border: none;
+  }
+`
+
+class BurgerMenu extends React.Component {
+  state = {
+    menuOpen: false,
+  }
+  render() {
+    const { menuItems } = this.props
+    return (
+      <React.Fragment>
+        <ButtonAsIconWrapper
+          data-test-id="burger-menu-expand"
+          onClick={() => this.setState({ menuOpen: true })}
+        >
+          <Icon iconName="Menu" overrideName="@pubsweet-pending.AppBar.Menu" />
+        </ButtonAsIconWrapper>
+        <ModalOverlay open={this.state.menuOpen} transparentBackground>
+          <MenuPanel>
+            <MenuItem p={3}>
```

Comment:
Since you've got `p={3}` in a couple of places (here and line 50), perhaps it's worth popping this in via styled components.

i.e. line 18:
```js
const MenuItem = styled(Box).attrs({p:3})`
```

Not super important, but could be nice in the unlikely event that someone might want to add any extra `MenuItem`s?